# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../imagemagick/multilib-fix.patch
# Copyright (C) 2008 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


On 64bit multilib-systems this had caused linking to 32bit libs.
	-- jsaw
re-diffed for 6.2.8 and then for 6.3 version - Aldas (Baldzius)

diff -Nur ImageMagick-6.3.8-orig/configure.ac ImageMagick-6.3.8/configure.ac
--- ImageMagick-6.3.8-orig/configure.ac	2008-02-18 11:50:29.000000000 +0100
+++ ImageMagick-6.3.8/configure.ac	2008-02-18 11:52:24.000000000 +0100
@@ -2170,7 +2170,7 @@
             CPPFLAGS="$CPPFLAGS -I${xml2_prefix}/include/libxml2"
         fi
         if test "${xml2_prefix}" != '/usr'; then
-            LDFLAGS="$LDFLAGS -L${xml2_prefix}/lib"
+            LDFLAGS="$LDFLAGS"
         fi
     fi
     failed=0;
